benchmarks/benchmarks_for_map.py

```diff
@@ -0,0 +1,33 @@
+# Initial benchmarks
+# See "Writing benchmarks" in the asv docs for more information.
+
+import sunpy.data.sample
+import sunpy.map
+
+
+class TimeSuite:
+    """
+    An example benchmark that times the performance of sunpy map
+    creation
+    """
+    def setup(self):
+        self._map = sunpy.data.sample.AIA_171_IMAGE
+
```

Comment:
Is there any reason to do this in a setup method? (other than just to show how it's done)? there isn't any compute being done here as the sample data would be downloaded on import of this file?

Reply:
Yes, it is recommended for timing benchmarks to take out of bounds all the preparations, to test only timing of function call

Reply:
But it is possible that I misunderstood mechanism of using sample data in general

Reply:
So the reason I don't think this needs to be in setup is all `sunpy.data.sample.AIA_171_IMAGE` is is a string representing a file path. There is no time or memory cost to this, so why don't we just use the `AIA_171_IMAGE` constant in the benchmarks?